### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hanywang2**
Updated README.md with contributors of different developers

### Summary 
The only change is the README.md where the contributors of CoursePlan from FA19, SP20, and FA20 have been added

### Testing Plan
Please make sure that everyone who has ever touched CoursePlan is included in the list

### Notes
This PR does not edit any code and therefore should not break trunk

### CODE REVIEW

**Review by hahnbeelee — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -22,9 +22,45 @@ Then access http://localhost:8080/
...
+- **Kathleen Xu** - PMM
+- **Will Spencer** - TPM
+- **Han Wang** - PM
+
+### FA19
+
+- **Theresa Cho** - Developer
+- **Ayesha Gagguturi** - Developer
+- **Bryant Lee** - Developer
+- **Ein Chang** - Developer/Designer
```
Comment (willespencer) [inline README.md:57]: Ein was only a designer first semester

File: README.md [hunk 2]
```diff
@@ -22,9 +22,45 @@ Then access http://localhost:8080/
...
+- **Emily Chan** - Designer
+- **Yuxuan Chen** - Designer
+- **Shane Yun** - Designer
+- **Chris Lee** - Designer
+- **Kathleen Xu** - PMM
+- **Will Spencer** - TPM
+- **Han Wang** - PM
+
+### FA19
```
Comment (willespencer) [inline README.md:52]: Lol can you add me :)

**Review by willespencer — CHANGES_REQUESTED** [review]
Just a couple edits but otherwise looks good! Thanks for doing this 😄

File: README.md (hunk 2 above)
Comment (hanywang2) [reply README.md:52]: That's what you get for not being memorable

Comment (hanywang2) [reply README.md:52]: Looool sorry man 🥺

**Review by willespencer — APPROVED** [review]

**Pull request merged**